openedx_proversity_reports/reports/learning_tracker_report.py

```diff
@@ -154,6 +155,22 @@ def _get_time_bewteen_sessions(self, user):
         except ValueError:
             return 0
 
+    def _get_total_sessions(self, user):
+        """
+        Calculate learner metrics for "Total of sessions".
```

Comment:
This method does not calculate anything, this method only returns the value. @diegomillan

Reply:
@Squirrel18 Done